legal-api/src/legal_api/services/business_details_version.py

```diff
@@ -308,33 +311,36 @@ def get_office_revision(filing, business) -> dict:  # pylint: disable=too-many-l
             .filter(office_history.deactivated_date == None)  # noqa: E711,E501;
         )  # pylint: disable=singleton-comparison
 
-        valid_office_types = current_types.union(historical_types).distinct().all()
+        valid_office_types = (
+            current_types.union(historical_types).distinct().all()
+        )  # result is like [("businessOffice", )]
 
         address_history = history_cls(Address)
-        for valid_office_type in valid_office_types:
-            office, _ = (
-                offices_current.filter(Office.office_type == valid_office_type.office_type)
-                .union(offices_historical.filter(office_history.office_type == valid_office_type.office_type))
+        add_history_columns = VersionedBusinessDetailsService.select_revision_columns(address_history)
+        for item in valid_office_types:
+            valid_office_type = item[0]
+            office = (
+                offices_current.filter(Office.office_type == valid_office_type)
+                .union(offices_historical.filter(office_history.office_type == valid_office_type))
                 .first()
             )
 
             offices_json[office.office_type] = {}
 
             addresses_current = (
-                db.session.query(Address, null().label("changed"))
+                db.session.query(Address)
                 .filter(Address.change_filing_id <= filing_id)
                 .filter(Address.office_id == office.id)
             )
 
             addresses_historical = (
-                db.session.query(address_history)
+                db.session.query(*add_history_columns)
                 .filter(address_history.change_filing_id <= filing_id)
                 .filter(address_history.office_id == office.id)
             )
 
-            addresses_result = addresses_current.union(addresses_historical).all()
-            for address_result in addresses_result:
-                address, _ = address_result
+            addresses = addresses_current.union(addresses_historical).all()
```

Comment:
`FM1000406`, filing_id: `146925`
In this case, business_id is 691 ( alternate_name_id )

But, there are no records for this business in Address and Address_history table ( my local DB )
So, it returns `None`, and will make the error